test/e2e/run-canary-test.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# This script runs e2e tests on the AWS Load Balancer Controller
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+echo "Running AWS Load Balancer Controller e2e tests with the following variables
+KUBE CONFIG: $KUBE_CONFIG_PATH
+CLUSTER_NAME: $CLUSTER_NAME
+REGION: $REGION
+OS_OVERRIDE: $OS_OVERRIDE"
+
+if [[ -z "${OS_OVERRIDE}" ]]; then
+  OS_OVERRIDE=linux
+fi
+
+GET_CLUSTER_INFO_CMD="aws eks describe-cluster --name $CLUSTER_NAME --region $REGION"
+
+if [[ -z "${ENDPOINT}" ]]; then
+  CLUSTER_INFO=$($GET_CLUSTER_INFO_CMD)
+else
+  CLUSTER_INFO=$($GET_CLUSTER_INFO_CMD --endpoint $ENDPOINT)
+fi
+
+VPC_ID=$(echo $CLUSTER_INFO | jq -r '.cluster.resourcesVpcConfig.vpcId')
+ACCOUNT_ID=$(aws sts get-caller-identity | jq -r '.Account')
+
+echo "VPC ID: $VPC_ID"
+
+eksctl utils associate-iam-oidc-provider \
```

Comment:
can this be done in the centralized image by default?

Reply:
We have similar script for other components as well and those scripts are reused to run tests, we can keep this in default image but then other component scripts will have to be modified as well. So kept this code here to have consistent pattern.